src/main/shortcutHandler.js

```diff
@@ -41,6 +41,9 @@ class Keybindings {
       ['editCopyAsMarkdown', 'CmdOrCtrl+Shift+C'],
       ['editCopyAsPlaintext', 'CmdOrCtrl+Shift+V'],
       ['editSelectAll', 'CmdOrCtrl+A'],
+      ['editDuplicate', 'Shift+CmdOrCtrl+P'],
+      ['editCreateParagraph', 'Shift+CmdOrCtrl+N'],
+      ['editDeleteParagraph', 'Shift+CmdOrCtrl+D'],
```

Comment:
Please add the shortcuts to the [documentation](https://github.com/marktext/marktext/blob/master/doc/KEYBINDINGS.md).